submissions/2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point/generate.py

```diff
@@ -0,0 +1,97 @@
+from collections import defaultdict
+
+from openff.qcsubmit.common_structures import Metadata, QCSpec, MoleculeAttributes, SCFProperties
+from openff.qcsubmit.datasets import BasicDataset
+from openff.qcsubmit.results import OptimizationResultCollection
+from openff.qcsubmit.results.filters import ConnectivityFilter, ConformerRMSDFilter
+from qcelemental.models import DriverEnum
+from qcelemental.models.results import WavefunctionProtocolEnum
+from qcportal import FractalClient
+
+
+def main():
+
+    optimization_results = OptimizationResultCollection.from_server(
+        client=FractalClient(),
+        datasets=["OpenFF ESP Industry Benchmark Set v1.0"],
+        spec_name="HF/6-31G*"
+    )
+    filtered_records = optimization_results.filter(
+        ConnectivityFilter(),
+        ConformerRMSDFilter(rmsd_tolerance=0.05)
+    ).to_records()
+
+    dataset = BasicDataset(
+        dataset_name="OpenFF ESP Industry Benchmark Set v1.1",
+        dataset_tagline="HF/6-31G* conformers of public industry benchmark molecules.",
+        description="A dataset containing molecules from the `OpenFF Industry Benchmark "
+        "Season 1 v1.1` with conformers taken from the "
+        "`OpenFF ESP Industry Benchmark Set v1.0` optimization (HF/6-31G*) set and with "
+        "wavefunctions stored and any conformers that underwent a connectivity change "
+        "filtered out.",
+        driver=DriverEnum.energy,
+        metadata=Metadata(
+            submitter="simonboothroyd",
+            long_description_url=(
+                "https://github.com/openforcefield/qca-dataset-submission/tree/master/"
+                "submissions/"
+                "2022-02-02-OpenFF-ESP-Industry-Benchmark-Set-v1.1-single-point"
+            )
+        ),
+        qc_specs={
```

Comment:
```suggestion
        qc_specifications={
```

Reply:
@SimonBoothroyd it looks like the field name is incorrect.

Reply:
Ah great catch!